### ISSUE TITLE
At least basic properties of a rack should be editable

### ISSUE THREAD

**Issue by tnozicka** [scylladb/scylla-operator#415]
**Is this a bug report or feature request?**
* Feature Request

**What should the feature do:**
At some point admin will want to edit e.g. cpu request/limit from 1 to 2, the answer he currently gets is
```
admission webhook "webhook.scylla.scylladb.com" denied the request: rack manager-rack: changes in resources are not currently supported
```

**Comment by munjalpatel** [comment]
I agree. I have also been burned by this where I had to simply start a new server to update resource request / limits.

**Comment by rogaha** [comment]
+1

**Comment by ronanbarrett** [comment]
Until this is implemented is there a way of vertically scaling the ScyllaCluster without downtime of the DB i.e. scaling up the pod resources without adding new members/nodes
